internal/api/grpc/server/middleware/call_interceptor.go

```diff
@@ -0,0 +1,16 @@
+package middleware
+
+import (
+	"context"
+
+	"google.golang.org/grpc"
+
+	"github.com/zitadel/zitadel/internal/api/call"
+)
+
+func CallHandler() grpc.UnaryServerInterceptor {
```

Comment:
CallHandler sounds very generic. What about something like ContextDurationHandler?

Reply:
used `CallDurationHandler` i think that describes it explicitely